### ISSUE TITLE
Completed Reviews Appear on Team Member Dashboard

### ISSUE THREAD

**Issue by fourthletter** [OpenTechFund/hypha#1693]
- Account name or name of person: Di

**Describe the bug**
Each team member's dashboard should only contain items that NEED a review. In other words, applications that need the staff member to submit a review. There is a glitch because items that have been completed remain on the dashboard.

**To Reproduce**
1. Go to dashboard 

**Expected behavior**
Applications should no longer appear on the dashboard once the staff member has submitted a review.

**Priority**
- Medium (slowing you down)

**Affected roles**
- Staff

**Comment by frjo** [comment]
@di I can not replicate this when testing in my local environment.

**Comment by fourthletter** [comment]
It's difficult to replicate on the test environment.

**Should my dashboard contain reviews I've completed?**

My dashboard when I click to expand my dashboard to show all submissions with reviews I've completed:
![image](https://user-images.githubusercontent.com/20019656/69060565-89a9f880-09e5-11ea-9a4a-d62666e7e792.png)


My dashboard shows submissions that need my review:

![Screenshot 2019-11-18 at 9 19 27 AM](https://user-images.githubusercontent.com/20019656/69060379-3cc62200-09e5-11ea-8a8c-d7532d208291.png)

**Comment by frjo** [comment]
On the dashboard there are one table "Your previous reviews" and one "Waiting for your review". They seem to work as they should when I test.

**Comment by fourthletter** [comment]
Thank you for clarifying. I found a similar scenario on the test site. The distinction between awaiting reviews and previous reviews isn't clear. 

For example, Testing a project as AC member test [https://test-apply.opentech.fund/apply/submissions/10/] has been completed. I submitted a review, but it's still on my dashboard under "waiting review."

Here is a link and screenshot: 

https://test-apply.opentech.fund/dashboard/#submissions-awaiting-review

![image](https://user-images.githubusercontent.com/20019656/69064308-f4f6c900-09eb-11ea-974c-7cdcf7157f1c.png)
